bitnami/pgpool/4/debian-12/rootfs/opt/bitnami/scripts/pgpool/run.sh

```diff
@@ -22,7 +22,7 @@ eval "$(pgpool_env)"
 eval "$(ldap_env)"
 
 command="$(command -v pgpool)"
-flags=("-n" "--config-file=${PGPOOL_CONF_FILE}" "--hba-file=${PGPOOL_PGHBA_FILE}")
+flags=("-n" "-D" "--config-file=${PGPOOL_CONF_FILE}" "--hba-file=${PGPOOL_PGHBA_FILE}")
```

Comment:
```suggestion
flags=("-n" "--config-file=${PGPOOL_CONF_FILE}" "--hba-file=${PGPOOL_PGHBA_FILE}")
is_boolean_yes "$PGPOOL_DISCARD_STATUS" && flags+=("-D")
```

Reply:
I agree that introducing a PGPOOL_DISCARD_STATUS environment variable with a default value of yes would provide the necessary flexibility while addressing the issue.